lib/desktop/init.dart

```diff
@@ -145,38 +141,42 @@ Future<Widget> initialize(List<String> argv) async {
       fingerprintProvider.overrideWithProvider(desktopFingerprintProvider),
       credentialProvider.overrideWithProvider(desktopCredentialProvider),
       clipboardProvider.overrideWith(
-            (ref) => ref.watch(desktopClipboardProvider),
+        (ref) => ref.watch(desktopClipboardProvider),
       ),
       supportedThemesProvider.overrideWith(
-            (ref) => ref.watch(desktopSupportedThemesProvider),
+        (ref) => ref.watch(desktopSupportedThemesProvider),
       )
     ],
     child: YubicoAuthenticatorApp(
       page: Consumer(
-        builder: ((_, ref, child) {
+        builder: ((context, ref, child) {
           // keep RPC log level in sync with app
           ref.listen<Level>(logLevelProvider, (_, level) {
-            rpc.setLogLevel(level);
+            ref.read(rpcProvider).valueOrNull?.setLogLevel(level);
```

Comment:
I'm also wondering if this should be changed to `requireValue` as well?

Reply:
No, in this case it is correct. We don't want to fail if this happens before the rpcProvider is initialized, so null here is fine (we'll just not call setLogLevel). `requireValue` would thrown an exception in that case.

Reply:
Alright, cool!